versionist.conf.js

```diff
@@ -1,4 +1,4 @@
-var _ = require('lodash');
+var _ = require(`${__dirname}/node_modules/lodash`);
```

Comment:
Yes, possibly, but given I've currently a broken published version, I'd really like to fix it.